apps/studio/prisma/schema.prisma

```diff
@@ -46,16 +56,25 @@ model Resource {
 
   permission Permission[]
 
-  mainBlob   Blob?   @relation("MainBlob", fields: [mainBlobId], references: [id])
-  mainBlobId BigInt? @unique
+  versionId BigInt?
+  version   Version? @relation(fields: [versionId], references: [id])
```

Comment:
issue: I'm not sure if the name `version` here would be confusing, because we are actually referring to the latest published version of that resource, not a relation to all the possible versions (that information is maintained within the versions table).

Reply:
Will rename to publishedVersionId